### PR TITLE
Closes #5 #11 - Improved how observers are configured and made scripts p...

### LINKED ISSUE #5: Improve how observers are configured
Currently, the command `fluo-dev configure fluo` generates a fluo.properties file.  When it does, it just a hard codes the stress test observers.  It would be better if the generated fluo.properties was concatenated with a `observer.props` file specified by the user.  A `observer.props.example` could be configured for the stress tests.

### PR DESCRIPTION
**Pull request by mikewalch**
...oint to new fluo-stress repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,26 +74,57 @@ data and restart them (if they are running).
...
 ```
 vim conf/fluo/fluo.properties
 ```
 
-Finally, deploy Fluo which will remove any existing install, rebuild fluo, install fluo, and 
-configure it using `conf/fluo/fluo.properties`.
+If you want to run Fluo with observers, avoid configuring them in `fluo.properites`.  Instead, you
+should create a file called `observer.props` in `conf/` by copying the example:
+```
+cp conf/observer.props.example conf/observer.props
```
Comment (keith-turner) [inline README.md]: Once we have the quick start example working w/ beta, we may want to consider making it the example.

File: README.md [hunk 2]
```diff
@@ -74,26 +74,57 @@ data and restart them (if they are running).
...
+The observer jar for [fluo-stress][stress] can be obtained by cloning and building its repo using the
+the steps below:
+```
+git clone https://github.com/fluo-io/fluo-stress.git
+cd fluo-stress/
+mvn package
+ls target/fluo-stress-*.jar
+```
+
+Copy your observer JAR for your application to `conf/fluo/observers`:
```
Comment (keith-turner) [inline README.md]: A sentence explaining what to do if you later update the observer jar would be nice.

File: README.md [hunk 3]
```diff
@@ -74,26 +74,57 @@ data and restart them (if they are running).
...
+Confirm that everything started by checking the monitoring pages of Hadoop & Accumulo:
+ * [Hadoop NameNode](http://localhost:50070/)
+ * [Hadoop ResourceManager](http://localhost:8088/)
+ * [Accumulo Monitor](http://localhost:50095/)
+
+Running Fluo
+------------
+
+With its dependencies running, Fluo can now be started.  Before starting Fluo, you can optionally
+verify or change configuration in the `fluo.properties` file generated by `fluo configure`.
```
Comment (keith-turner) [inline README.md]: shouldn't this be : generated by `fluo-dev configure`  ?

File: README.md (hunk 1 above)
Comment (mikewalch) [inline README.md]: sounds good to me

File: README.md (hunk 2 above)
Comment (mikewalch) [inline README.md]: added in c51ff8186b4

File: README.md (hunk 3 above)
Comment (mikewalch) [inline README.md]: added in c51ff8186b4

**Comment by keith-turner** [comment]
+1

**Pull request merged**